feat: :sparkles: data selection and preservation (#134)
onlimit-study/dmp: sections/selection.qmd

    development, to ensure continuity beyond the funding period.
-   Biobank samples: We will initially retain biological samples until
    the end of the current funding period. Plans are in place to
    establish a Steno Biobank before this point, where we will transfer
    ON LiMiT samples for extended preservation.

## Study timeline

Data collection for the feasibility study will span approximately 13
months (12 months per participant), while the main study will involve a
two-year participation window per participant, with the last
participant's final visit expected in 2032. Retention commitments extend
well beyond this timeline to support future research.

## Access and future use

To maximise reuse, we will develop a searchable website to enable
discovery of available variables and datasets, as described in the
[Documentation and metadata section](sections/metadata.qmd). Initially,
access is expected to be of particular interest to the study team. In
accordance with the Publications Guidance, data will be made available
to a wider research community once primary findings have been published,
subject to an application and approval process.

> 

Reviewer: ```suggestion
```

Remove empty lines.

    development, to ensure continuity beyond the funding period.
-   Biobank samples: We will initially retain biological samples until
    the end of the current funding period. Plans are in place to
    establish a Steno Biobank before this point, where we will transfer
    ON LiMiT samples for extended preservation.

## Study timeline

Data collection for the feasibility study will span approximately 13
months (12 months per participant), while the main study will involve a
two-year participation window per participant, with the last
participant's final visit expected in 2032. Retention commitments extend
well beyond this timeline to support future research.

## Access and future use

To maximise reuse, we will develop a searchable website to enable
discovery of available variables and datasets, as described in the
[Documentation and metadata section](sections/metadata.qmd). Initially,
access is expected to be of particular interest to the study team. In
accordance with the Publications Guidance, data will be made available
to a wider research community once primary findings have been published,
subject to an application and approval process.

> <!--TODO Daniel to check if we can publish the publication guidance (and the correct name)-->

Author: ```suggestion

```

# Selection and preservation

The long-term preservation of research data and biological samples is
important to make sure we have reproducibility and compliance with
ethical and legal requirements. This section outlines the strategy for
selecting which data and samples will be retained, how it will be stored
and for how logn, and the mechanisms for ensuring secure storage and
accessibility. All processes adhere to the FAIR principles and comply
with GDPR to protect participant privacy and data integrity.

## Retention and storage

As a general rule, all data generated by or from participants will be
retained and stored on GenomeDK’s secure servers. Exceptions may include
more technical/system data such as log files, which have limited
long-term value.

-   Study Data: All core datasets will be preserved for a minimum of 15
    years after study completion. The long-term plan is to integrate
    these datasets into a joint Steno Database, currently under
    development, to ensure continuity beyond the funding period.

> -   Biobank Samples: Biological samples will initially be retained until

Reviewer: ```suggestion
-   Biobank samples: We will initially retain biological samples until
```

    the end of the current funding period. Plans are in place to
    establish a Steno Biobank before this point, where ON LiMiT samples
    will be transferred for extended preservation.

## Study timeline

Data collection for the feasibility study will span approximately 13
months (12 months per participant), while the main study will involve a
two-year participation window per participant, with the last
participant’s final visit expected in 2032. Retention commitments extend
well beyond this timeline to support future research.

<!--TODO Daniel to check if these timelines are sensible-->

## Access and future use

> To maximise reuse, a searchable website will be developed to enable

Reviewer: ```suggestion
To maximise reuse, we will develop a searchable website to enable
```

discovery of available variables and datasets, as described in the
[Documentation and metadata section](sections/metadata.qmd). Initially,
access is expected to be of particular interest to the study team. In
accordance with the Publications Guidance, data will be made available
to a wider research community once primary findings have been published,
subject to an application and approval process.

<!--TODO Daniel to check if we can publish the publication guidance (and the correct name)-->

> <!--TODO Consider if some curated data sets should be made available on Zenodo or the likes-->

Reviewer 2: We can take that in the future. Not sure others in the project would be too interested in that. Would be cool to make a simulated/synthetic dataset that students could use - also for testing their coding before getting access to the data on GenomeDK. We can leave this for now though.
